src/app/components/navbar/NavBar.test.js

```diff
@@ -63,6 +64,23 @@ describe("NavBar", () => {
             expect(component.find("Link[to='/florence/uploads/data']").exists()).toBe(false);
         });
 
+        describe("when enableNewSignIn feature flag is enabled", () => {
+            const props = {
+                ...defaultProps,
+                config: {
+                    ...defaultProps.config,
+                    enableNewSignIn: true,
+                },
+            };
+            const component = shallow(<NavBar {...props} user={authenticatedUser} />);
+            const nav = component.find(Link);
+            it("third item should be named 'Preview teams'", () => {
```

Comment:
do we need to be so specific it needs to be third? I don't see the assertion for this in the test anyway? you checking the class and loops thru its elements checking if they have a child of `NavbarItemsWithCognitoAuth` ?  I can't see where you checking if navigation has `Preview teams` item. Please make sure test are simple. 
Loops in test should be avoided if possible.